### PR TITLE
Update download_list.json

### PR DESCRIPTION
**Pull request by PRElias**
https://github.com/Francommit/win10_emulation_station/issues/27#issuecomment-626079421

### CODE REVIEW

File: download_list.json [hunk 1]
```diff
@@ -1,8 +1,8 @@
 {
 	"downloads": [
 	  {"file":"ePSXe205.zip", "url":"http://www.epsxe.com/files/ePSXe205.zip"},
-	  {"file":"pcsx2-1.6.0-setup.exe", "url":"http://delivery2.filecroco.com/kits_4/pcsx2-1.6.0-setup.exe"},
-	  {"file":"RetroArch.7z", "url": "https://buildbot.libretro.com/stable/1.8.6/windows/x86_64/RetroArch.7z"},
+	  {"file":"pcsx2-1.6.0-binaries.7z", "url":"https://pcsx2.net/download/releases/windows/send/40-windows/125-pcsx2-1-6-0-binaries.html"},
```
Comment (Francommit) [inline download_list.json:4]: If you're changing this to be a binary, you'll have to change the way it handles it in the script

File: download_list.json [hunk 2]
```diff
@@ -1,8 +1,8 @@
...
+	  {"file":"RetroArch.7z", "url": "https://buildbot.libretro.com/stable/1.6.7/windows/x86_64/RetroArch.7z"},
```
Comment (Francommit) [inline download_list.json:5]: Is there a reason we'd use 1.6.0 rather than 1.8.6 ?

File: download_list.json [hunk 3]
```diff
@@ -15,19 +15,19 @@
 	  {"file":"genesis_plus_gx_libretro.dll.zip", "url":"http://buildbot.libretro.com/nightly/windows/x86_64/latest/genesis_plus_gx_libretro.dll.zip"},
 	  {"file":"snes9x_libretro.dll.zip", "url":"http://buildbot.libretro.com/nightly/windows/x86_64/latest/snes9x_libretro.dll.zip"},
 	  {"file":"N-Warp Daisakusen V1.1.smc", "url":"https://github.com/OpenEmu/OpenEmu-Update/raw/master/Homebrew/SNES/N-Warp%20Daisakusen/N-Warp%20Daisakusen%20V1.1.smc"},
-	  {"file":"Marilyn_In_the_Magic_World_(010a).7z","url":"http://psxdev.net/homebrew/files/Marilyn_In_the_Magic_World_(010a).7z"},
+	  {"file":"marilyn_v0.12a.7z","url":"https://www.ps2-home.com/media/brew/marilyn_v0.12a.7z"},
```
Comment (Francommit) [inline download_list.json:18]: Nice find! Again though, we'll need to change the way the script handles the file name.

File: download_list.json (hunk 1 above)
Comment (PRElias) [reply download_list.json:4]: sorry man, I made a mistake, thanks for the tip
you can revert that if its possible

File: download_list.json (hunk 2 above)
Comment (PRElias) [reply download_list.json:5]: no really, it was just wjat I found in a hurry, sorry again

File: download_list.json (hunk 3 above)
Comment (PRElias) [reply download_list.json:18]: I think you can reject the pull and sorry for the trouble
I was looking to help but in a hurry and made some mistakes

Comment (Francommit) [reply download_list.json:18]: Hey PRElias - thanks for your help, I've made a bunch of changes and everything now works!

**Pull request closed without merging**